jacobbishop1000/video-crime-miner
# Mar31 demo, Batch Video Analysis Jobs, Job Completion Status Checking, and Sorting labels on file-rekognition-view page

Steps for seeing changes from user's perspective

1. log in to test user or your own user
2. create a new case
3. add at least two files to case
4. create a new label detection job, and use check boxes to select or deselect any files on the case
5. add desired terms, at least 2 to test one of the sorting methods later, and create the job
6. job view pop-up should load with the new batch job, should initially show 'Pending'
7. wait ~30ish seconds and the 'Pending' should go away
8. click on the batch job, should load file-rekognition-view page
9. should be able to select sorting type by either "File" or "Keyword", and select individual label detections to show on the video-player

The video player is a bit buggy switching between videos, sometimes you have to click it a second time

I may update the example inserts in db.sql so that whoever tests this won't have to go through the process of creating a case and uploading files. I've just been manually inserting columns into the File Table in our db with files that are already in the bucket, but the above steps will work if someone gets to it before I change the inserts.
